src/App.js

```diff
@@ -5,28 +5,26 @@ import AddTodoform from "./AddTodoForm";
 
 function App() {
 
-  const [todoList, setTodoList] = React.useState([]);
-  const [isLoading, setIsLoading] = React.useState(true);
+  const [todoList, setTodoList] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
   console.log(process)
-  React.useEffect(() => {
+  
+ useEffect(() => {
```

Comment:
fix both useEffect indentations. should be one place further than App function indentation